javascript-data-model/order-history.js

```diff
@@ -0,0 +1,53 @@
+var orderHistory = [
+  {
+    order_number: ['114-3941689-8772232'],
+    order_name: ['JavaScript for impatient programmers'],
+    order_placed: ['August 4, 2020'],
+    order_delivered: ['August 8, 2020'],
+    return_window: ['Sep 7, 2020'],
+    total_price: ['$34.00'],
+    listed_price: ['$31.55'],
+    author: ['Rauschmayer, Dr. Axel'],
+    ship_to: ['JS Masher'],
+    delivery: [null]
```

Comment:
Rather than separating everything into its own array, I suggest you make one array of objects, with each object representing a single item, and just have named properties for the data correlated to the order